day 1/exercise 3/solution.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import "fmt"
+
+type Employee interface {
+	findSalary() int
+}
+
+type FullTime struct {
+	daysWorked int
+}
+
+type Contractor struct {
+	daysWorked int
+}
+type Freelancer struct {
+	 hrsWorked int
+}
+func (f FullTime) findSalary() int {
+	return f.daysWorked*500
+}
+func (c Contractor) findSalary() int {
+	return c.daysWorked*100
+}
+func (fl Freelancer) findSalary() int {
+	return fl.hrsWorked*10
+}
+
+func main() {
+	fte:= FullTime{25}
+	cont:=Contractor{18}
+	freeLan:=Freelancer{44}
+	emp := []Employee{fte,cont,freeLan}
```

Comment:
where are considering 28 days in the solution. you need to calculate salary for 28 for FULL time employee right?